itest/rust/src/plane_test.rs

```diff
@@ -0,0 +1,166 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use std::fmt::Debug;
+
+use crate::itest;
+use godot::{
+    prelude::{inner::InnerPlane, real, Plane, RealConv, ToVariant, Vector3},
+    private::class_macros::assert_eq_approx,
+};
+
+#[itest]
+fn plane_equiv() {
```

Comment:
You have one epic `#[itest]` of 150 LoC 🙂 

Could you extract the local functions to a global level, and maybe split this into multiple `#[itest]` depending on what you check? That way, tests can succeed/fail independently, and it's easier to detect regressions. Each test also becomes easier to inspect if there is less code.

Reply:
There's no very strict rule. You can group them if things are closely related (e.g. multiple "overloads" or set/get pairs, etc). Otherwise, use multiple ones.